test/sanitytest.c

```diff
@@ -189,7 +189,9 @@ static int test_sanity_sleep(int i)
 #if defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200112L
     /* disarm the timer */
     do {
-        if (setitimer(ITIMER_REAL, NULL, NULL)) {
+        static const struct itimerval it = { { 0 } };
+
+        if (setitimer(ITIMER_REAL, &it, NULL)) {
```

Comment:
Suggest this change which makes the initialisation independent of the internal layout of the structure and avoids a global:

```suggestion
        struct itimerval it;

        memset(&it, 0, sizeof(it));
        if (setitimer(ITIMER_REAL, &it, NULL)) {
```

Alternatively, we could rely on the zero initialisation of globals mandated by the standard:
```suggestion
        static const struct itimerval it;

        if (setitimer(ITIMER_REAL, &it, NULL)) {
```

My preference is ever so slightly for the first.

Reply:
You're right, no explicit initialisation is needed here, will update the patch.